atomic_reactor/plugins/post_koji_upload.py

```diff
@@ -444,6 +444,25 @@ def add_log_type(output, arch):
 
         return output_files
 
+    def update_buildroot_koji(self, buildroot, output):
+        """
+        put the final koji information in the buildroot under extra.osbs
+        """
+        docker = output[1]['extra']['docker']
+
+        name = ''
```

Comment:
What causes  empty `name`? Failed build?

Reply:
I don't think you can get to this stage of koji_upload and not have a valid tag in a repo.  But I'd rather not break builds in case I'm wrong, so I'll set a sensible default.

Reply:
ok